### ISSUE TITLE
Error with latest update

### ISSUE THREAD

**Issue by lucasnguyen3979** [vanhbakaa/Seed-App-Mine-Seed-BOT-Telegram#6]
![image](https://github.com/user-attachments/assets/299d2325-fd1c-4cce-9b5c-3c43fa45d94e)

**Comment by vanhbakaa** [comment]
Update again i upload a new version after it

**Comment by lucasnguyen3979** [comment]
> Update again i upload a new version after it

Which version? I'm using this version:
![image](https://github.com/user-attachments/assets/141277d8-172f-4e03-bc89-8972417a45d4)

**Comment by vanhbakaa** [comment]
> ![image](https://private-user-images.githubusercontent.com/25925555/365350472-299d2325-fd1c-4cce-9b5c-3c43fa45d94e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU3MTMwNTgsIm5iZiI6MTcyNTcxMjc1OCwicGF0aCI6Ii8yNTkyNTU1NS8zNjUzNTA0NzItMjk5ZDIzMjUtZmQxYy00Y2NlLTliNWMtM2M0M2ZhNDVkOTRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA3VDEyMzkxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViNmIxODczOTVjYmE0ZjU5ZWVlMzRkNjkwMTU4NGEyMzdhOTVmYjYxZjc1YzIwNTk4YzQyMWI4NDEzNzNmMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SrZ3C95eDAFOlYO2mF3jY95duGvRv8tBqjBqi4xgS9g)

I already fix it in newest

**Comment by lucasnguyen3979** [comment]
> I already fix it in newest

Yes, I'm absolutely using the latest one and still have that error as screenshot.

**Comment by vanhbakaa** [comment]
it work fine for me. try to use python 3.11.5 same as mine

**Comment by vanhbakaa** [comment]
tell me if it work or not

**Comment by lucasnguyen3979** [comment]
> it work fine for me. try to use python 3.11.5 same as mine

Yep, it worked with version `3.11.5`...
